scripts/sexbound/lib/sexbound/actor.lua

```diff
@@ -642,14 +642,16 @@ function Sexbound.Actor:setup(actorConfig)
     if self._config.arousedStrong then actorStatus:addStatus("sexbound_aroused_strong") end
     if self._config.inHeat then actorStatus:addStatus("sexbound_aroused_heat") end
     if self._config.isDefeated then actorStatus:addStatus("sexbound_defeated") end
+    if self._config.birthcontrol then actorStatus:addStatus("birthcontrol") end
+    if self._config.canUseSexUIDefeated then actorStatus:addStatus("sexbound_defeated_can_use_ui") end
```

Comment:
Naming inconsistency with how it's named on the entity "can_use_sex_ui_defeated".

Reply:
I'll update that.